static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -372,6 +372,23 @@ function DropdownItem({
         <RecommendedItemTitle>{item.title}</RecommendedItemTitle>
       </RecommendedItem>
     );
+  } else if (item.type === ItemType.TAG_VALUE) {
```

Comment:
this seems to be mostly copied over from the else with a couple lines changes. could we just add the `documentation` field to `item` for these item types instead of having a new else if